ios/RCTConvert+BitmovinPlayer.swift

```diff
@@ -335,33 +335,62 @@ extension RCTConvert {
         }
     }
 
+    /**
+     Utility method to get a `DrmConfig` from a JS object.
+     - Parameter json: JS object
+     - Returns: The generated `DrmConfig` object
+     */
```

Comment:
Nit: IMO doc like this is not adding any value. No additional information besides what I see in the method signature. But fine for me too keep it as it aligns with the general practice in this file

Reply:
Could be useful for external or new contributors.
This was started by our initial contributor and we kept it consistent when taking over.